easybuild/tools/options.py

```diff
@@ -213,6 +213,9 @@ def override_options(self):
             'cleanup-builddir': ("Cleanup build dir after successful installation.", None, 'store_true', True),
             'cleanup-tmpdir': ("Cleanup tmp dir after successful run.", None, 'store_true', True),
             'color': ("Allow color output", None, 'store_true', True),
+            "default-optimisation": ("Set default optimization level, overriding easybuild optimization default "
+                                     "(possibilities are: noopt, lowopt, defaultopt, opt)",
```

Comment:
hmm, we shouldn't hardcode the options here, we should try and obtain them via `easybuild.tools.toolchain.compiler`?

also, the option parser supports a `choice` type of option, so do:

```python
'default-opt-level': ("Specify default optimisation level", 'choice', 'store', 'defaultopt', known_opt_levels),
```

I like how you use different spelling for `optimisation` vs `optimization` in the option name and the help, so let's use `opt` instead ;)